source/MRViewer/ImGuiMenu.cpp

```diff
@@ -297,6 +297,7 @@ void ImGuiMenu::addMenuFontRanges_( ImFontGlyphRangesBuilder& builder ) const
 {
     builder.AddRanges( ImGui::GetIO().Fonts->GetGlyphRangesCyrillic() );
     builder.AddChar( 0x2116 ); // NUMERO SIGN (shift+3 on cyrillic keyboards)
+    builder.AddChar( 0x2212 ); // MINUS SIGN
```

Comment:
minus is standard sign, why load it explicitly

Reply:
This is a fancy long minus, looks better than the default ASCII one.